packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -5,27 +5,32 @@ const { toBeAccessible } = require('../commands/expect');
 const { getViewports, describeViewports, setViewport } = require('../commands/viewport-helpers');
 
 const hideInputCaret = require('../commands/hide-input-caret');
+const { element } = require('../commands/validates');
 
 class TerraService {
   constructor(options = {}) {
-    const { formFactor, theme } = options;
-
-    this.formFactor = formFactor;
-    this.theme = theme || 'terra-default-theme';
+    const theme = { theme: 'terra-default-theme' };
+    this.serviceOptions = { ...theme, ...options };
   }
 
   /**
    * Service hook executed prior to test execution.
    * Initializes the Terra Service's custom commands.
    */
   before(capabilities) {
+    this.serviceOptions = { ...this.serviceOptions, ...global.browser.config.serviceOptions };
```

Comment:
We should use a hook like `onPrepare` that returns the config option to set these options rather than reading from the global scope

Reply:
That makes sense: https://github.com/cerner/terra-toolkit/pull/464/commits/04718224294139afe8b92e05a16b2910ba550b35